ext_emconf.php

```diff
@@ -18,7 +18,7 @@
     'author_company'   => 'jambage.com',
     'constraints'      => [
         'depends'   => [
-            'php'   => '7.2.0-7.3.99',
+            'php'   => '7.2.0-',
```

Comment:
You must set an upper limit. This is necessary to tell other users that it has been tested with this version.